concordium-node/src/bin/cli.rs

```diff
@@ -138,10 +140,49 @@ async fn main() -> anyhow::Result<()> {
         None
     };
 
-    if let Some(ref import_path) = conf.cli.baker.import_path {
-        info!("Starting out of band catch-up");
-        consensus.import_blocks(import_path.as_bytes());
-        info!("Completed out of band catch-up");
+    // Out-of-band catch-up
+    if let Some(ref import_blocks_from) = conf.cli.baker.import_blocks_from {
+        let from: anyhow::Result<String> = if let Ok(import_url) = Url::parse(import_blocks_from) {
+            let default_filename = format!(
+                "{}_{}{}",
+                config::CATCHUP_FILE_BASENAME,
+                Utc::now().timestamp(),
+                config::CATCHUP_FILE_EXT
+            );
+            let filename = import_url
+                .path_segments()
+                .and_then(|x| x.last())
+                .map(|x| x.to_string())
+                .unwrap_or(default_filename);
```

Comment:
Sorry for not commenting this the first time around, but I don't think the URL should be used to construct the file name.
Always using the `default_filename` is more predictable.

Reply:
Deferred to #437